network/wsNetwork_test.go

```diff
@@ -1411,11 +1405,9 @@ func TestPeeringWithIdentityChallenge(t *testing.T) {
 	// the underlying connection is being closed. In this case, the read loop
 	// on the peer will detect and close the peer. Since this is asynchronous,
 	// we wait and check regularly to allow the connection to settle
-	assert.Eventually(
-		t,
-		func() bool { return len(netB.GetPeers(PeersConnectedIn)) == 1 },
-		5*time.Second,
-		100*time.Millisecond)
+	assert.Eventually(t, func() bool {
+		return len(netB.GetPeers(PeersConnectedIn)) == 1
+	}, time.Second, 50*time.Millisecond)
```

Comment:
is this timeout long enough on a build server? (Eventually is checking constantly, so this isn't a minimum duration, just a maximum)

Reply:
Answered below, all set.